src/vs/workbench/contrib/chat/browser/chatContentParts/chatReferencesContentPart.ts

```diff
@@ -392,8 +403,11 @@ class CollapsibleListRenderer implements IListRenderer<IChatCollapsibleListItem,
 		}
 
 		if (this.menuId) {
+			if (data.state !== undefined) {
+				chatEditingWidgetFileStateContextKey.bindTo(templateData.contextKeyService).set(data.state);
```

Comment:
You'll want to reset this in the other case, because the template will be reused